easybuild/easyconfigs/s/scenicplus/scenicplus-1.0a1-18072024-foss-2023a.eb

```diff
@@ -0,0 +1,281 @@
+easyblock = 'PythonBundle'
+
+name = 'scenicplus'
```

Comment:
@pavelToman `SCENIC+` is official software name, so we should use that.

The `+` is annoying though, so let's go with `SCENISplus` as a compromise?